adafruit_gc_iot_core.py

```diff
@@ -26,8 +26,19 @@
   https://github.com/adafruit/Adafruit_CircuitPython_Logging
 
 """
+
 import time
 
+try:
+    from typing import Any, Callable, Dict, Optional, Type, Union
+    from types import TracebackType
+
+    # pylint: disable=unused-import
```

Comment:
It looks like this is unused because you're using quoted typing for this.  You should be able to switch to using it unquoted, was there some sort of CI issue?

Reply:
Not with CI, but Pylance does not like it. Commenting out the last two imports there results in the following:
![image](https://user-images.githubusercontent.com/23083317/174631921-00d1bc11-d650-451e-b5a3-7fc6fa15b4bb.png)

Reply:
Sorry, I misread your question. One sec

Reply:
It should be added to `setup.py` and `requirements.txt`, let me know if you need help with this.  The other Adafruit libraries you're using for typing will also need to be added there as well.

Reply:
Fixed this, sorry for not getting back. I'm not sure what the issue was. pip was reporting that the library was installed, but there was nothing in site-packages. The ``.dist-info`` folder was there which really confused me as well.

Getting rid of that folder and trying again seemed to do the trick, so I should be good. I'll fix the CI issue below and then everything should be ready.

Reply:
Oh gotchya, yeah, I actually had something exactly like this happen to me and it took me forever to realize what it was. @____@ Glad it got fixed!